src/types/guild.ts

```diff
@@ -547,9 +547,11 @@ export interface PrunePayload {
 }
 
 export interface PruneOptions {
-  /** number of days to count prune for (1 - 30). Defaults to 7 days. */
+  /** Number of days to prune (1-30). Default: 7 */
   days: number;
-  /** Include members with these role ids */
+  /** Whether 'pruned' is returned, discouraged for large guilds. Default: true */
+  compute_prune_count: string[];
+  /** Role(s) to include */
   roles: string[];
```

Comment:
shouldnt this be optional? im not sure

Reply:
@Skillz4Killz it is not mentioned in the documentation, I assume it is optional.

Reply:
Actually all options should be optional

Reply:
Any reference about whether they should be optional or not?

Reply:
So Discord specifies default values in the docs so all should be optional
`include_roles`:
> You can optionally include specific roles in your prune by providing the `include_roles` parameter

Reply:
> So Discord specifies default values in the docs so all should be optional
> `include_roles`:
> 
> > You can optionally include specific roles in your prune by providing the `include_roles` parameter

That refers to include_role field, not `compute_prune_count`